kusto/trustedEndpoints/wellKnownKustoEndpoints.go

```diff
@@ -0,0 +1,90 @@
+package truestedEndpoints
```

Comment:
So we weren't on the same page about it.. have the normal json file here and use https://pkg.go.dev/embed